### PR TITLE
Signup copy adjustment

### PR DESCRIPTION
**Pull request by st0nebraker**
Per design [spec](https://docs.google.com/document/d/1o2BVLvqwq3BPbj6jC7IXTbriu2SPadCe7wbeBsgjX3k/edit#) (2nd row)

## App preview:

- [Web](https://sg-web-becca-signup-copy-fix.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (0.00 kb) | 0.00% (0.00 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/becca/signup-copy-fix/compare-report.html;tab=live_object) for a full comparison between the commits 9e43247 and ded8db9 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/web/src/auth/CloudSignUpPage.tsx [hunk 1]
```diff
@@ -167,7 +167,7 @@ export const CloudSignUpPage: React.FunctionComponent<React.PropsWithChildren<Pr
 
                     {invitedBy ? 'With a Sourcegraph account, you can:' : 'With a Sourcegraph account, you can also:'}
                     <ul className={styles.featureList}>
-                        <li>Search across all your public repositories</li>
+                        <li>Search across +2M open source repositories</li>
```
Comment (sqs) [inline client/web/src/auth/CloudSignUpPage.tsx:170]: The + being before the number seems weird to me. "Search across two million plus open source repositories" is how you'd say it, so the plus comes after 2M.

Comment (st0nebraker) [reply client/web/src/auth/CloudSignUpPage.tsx]: Sure thing, I agree, wasn't sure if this was some pattern/convention. Changed it around 👍🏻

**Comment by st0nebraker** [comment]
Quick stamp please @taylorsperry  ?

**Review by taylorsperry — APPROVED** [review]
LGTM

**Pull request merged**